### PR TITLE
Add area owners responsibilities

### PR DESCRIPTION
**Pull request by prophile**

### CODE REVIEW

File: docs/programme/area-owners.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+* Live stream
+* Live stream tech
+* Live stream commentators
+* Virtual competition broadcast
+
+## Production
+
+Responsibilities:
+
+* Prizegiving ceremony
```
Comment (PeterJCLaw) [inline docs/programme/area-owners.md:104]:
```suggestion
* Prize-giving ceremony
```

File: docs/programme/area-owners.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+* Feedback & surveys
+
+## Kit
+
+Responsibilities:
+
+* Kit handout
+* Replacements
+* Documentation
+* Handback/recovery/return
```
Comment (PeterJCLaw) [inline docs/programme/area-owners.md:69]:
```suggestion
* Hand-back/recovery/return
```

File: docs/programme/area-owners.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+* Blog posts/announcements
+* Social media (incl. competitor social media)
+* Inviting guests
+* Arranging guest speaker
+* Swag
+* Competition booklet
+* Wristbands/lanyards
+* Reception
+* Media consent arrangements
+* Website (NB srcomp)
```
Comment (PeterJCLaw) [inline docs/programme/area-owners.md:36]: What does "NB srcomp" mean here?

File: docs/programme/area-owners.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+## Volunteer coördination
+
+Responsibilities:
+
+* Roles
+* Getting volunteers to events
+* Recruiting
+* Instructing volunteers
+* Inductions
+* Volunteer wellbeing (incl. breaks, refreshments)
```
Comment (PeterJCLaw) [inline docs/programme/area-owners.md:134]:
```suggestion
* Volunteer well-being (incl. breaks, refreshments)
```

File: docs/programme/area-owners.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+Responsibilities:
+
+* Judging
+* Match operations
+* Score entry
+* srcomp
+* Scheduling (match and day)
+* Rules
+
+## Volunteer coördination
```
Comment (PeterJCLaw) [inline docs/programme/area-owners.md:125]: Please add "coördination" to the `.spelling` file in a suitable section.

File: docs/programme/area-owners.md (hunk 3 above)
Comment (prophile) [reply docs/programme/area-owners.md:36]: Excellent question, this is a literal copy from the source. Will expand but: note that there are interactions with srcomp for the competition website, and srcomp is ostensibly within the remit of Game.

File: docs/programme/area-owners.md (hunk 1 above)
Comment (prophile) [reply docs/programme/area-owners.md]: ecae560b0b4158ed7d10c151035464df2857a97a

File: docs/programme/area-owners.md (hunk 5 above)
Comment (prophile) [reply docs/programme/area-owners.md:125]: ecae560b0b4158ed7d10c151035464df2857a97a

File: docs/programme/area-owners.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+* Getting volunteers to events
+* Recruiting
+* Instructing volunteers
+* Inductions
+* Volunteer well-being (incl. breaks, refreshments)
+* Volunteer rest area
+* Volunteer meals (end of day)
+* Bins
+* Sanitation
+* Cleaning
```
Comment (raccube) [inline docs/programme/area-owners.md:139]: Makes this feel more like Volunteer Services

**Review by raccube — APPROVED** [review]

File: package-lock.json [hunk 7]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "runbook",
+  "name": "sr-runbook",
```
Comment (PeterJCLaw) [inline package-lock.json:2]: This seems unrelated?

File: docs/programme/area-owners.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+Responsibilities:
+
+* In schools
+* Helpdesk
+* First-line support
+* Roving help at competition
+* Microgames
+* Discord bot & moderation
+* Tech day content/structure
+* Power tools area
```
Comment (WillB97) [inline docs/programme/area-owners.md:50]: Why did this end up under mentoring?

Comment (prophile) [reply docs/programme/area-owners.md:50]: Goooood question. I think because it ends up being more instruction than supervision?

Comment (prophile) [reply docs/programme/area-owners.md:50]: Where do you think it should live?

File: docs/programme/area-owners.md (hunk 6 above)
Comment (prophile) [reply docs/programme/area-owners.md]: 339801a073b980074e96506da561b39bd87730b5

File: package-lock.json (hunk 7 above)
Comment (prophile) [reply package-lock.json:2]: This is `main` coming in I think

Comment (PeterJCLaw) [reply package-lock.json:2]: Is your local directory called `sr-runbook` by chance?

File: docs/programme/area-owners.md (hunk 8 above)
Comment (WillB97) [reply docs/programme/area-owners.md:50]: Probably H&S as this is mainly about using tools safely.

File: package-lock.json (hunk 7 above)
Comment (prophile) [reply package-lock.json:2]: Ah yes, iunno how this got in, will fix

File: docs/programme/area-owners.md (hunk 8 above)
Comment (prophile) [reply docs/programme/area-owners.md:50]: Will move

File: docs/programme/area-owners.md (hunk 3 above)
Comment (PeterJCLaw) [reply docs/programme/area-owners.md]: https://github.com/srobo/runbook/pull/176/commits/d943e4b2faf4b11d1bd3312689a75a62833f836a

File: docs/programme/area-owners.md (hunk 8 above)
Comment (prophile) [reply docs/programme/area-owners.md]: c170a7961665a91ee83036394fd8da8dfa730578

File: package-lock.json (hunk 7 above)
Comment (PeterJCLaw) [reply package-lock.json]: https://github.com/srobo/runbook/pull/176/commits/76d13a7adb5e555cfe8a2e15163ec2ae893b409a
